Formula/d/doxymacs.rb

```diff
@@ -50,11 +50,11 @@ def install
   end
 
   test do
-    (testpath/"test.el").write <<~EOS
+    (testpath/"test.el").write <<~'LISP'
       (add-to-list 'load-path "#{elisp}")
```

Comment:
Need to keep interpolation (non-quoted heredoc) to expand `#{...}`

Reply:
indeed